Source/WTF/wtf/threads/Signals.h

```diff
@@ -124,18 +124,36 @@ struct SignalHandlers {
 // and once commited they can't be turned off.
 WTF_EXPORT_PRIVATE void addSignalHandler(Signal, SignalHandler&&);
 WTF_EXPORT_PRIVATE void activateSignalHandlersFor(Signal);
-
+WTF_EXPORT_PRIVATE void finalizeSignalHandlers(void);
```

Comment:
You need to add a definition of this to `SignalsWin.cpp` for the windows build. Also `(void)` -> `()`